hack/build/ci/update-renovate-json5.py

```diff
@@ -0,0 +1,24 @@
+import json5
```

Comment:
please add json5 to requirements.txt somewhere as dependency.

Reply:
it;s in readme https://github.com/dynatrace/dynatrace-operator/blob/04b27c3b322371b5ddd7b7ac17449f3ab80514bb/hack/doc/README.md#L16-L23

Reply:
Does it make sense to add it there as it is only a description for the python scripts in the doc folder.

Maybe we should create a real `requirements.txt` in the hack folder that includes our 3 requirements: 
```
argparse==1.4.0
pyyaml==6.0.1
json5==0.9.24
```

Reply:
Done 👍